{% extends "pred/base_layout.html" %}

{%block content_right_panel %}
    {% if username %}
      {% include "pred/user_header.html" %}
    {% endif %}
  <h2>Help</h2>
  <table cellspacing=0 cellpadding=0 border=0>
    <tr>
      <td class="Greylink"><a href="#1">1.&nbsp;Summary<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      <td class="Greylink"><a href="#2">2.&nbsp;Usage<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      <td class="Greylink"><a href="#3">3.&nbsp;Output<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      <td class="Greylink"><a href="#4">4.&nbsp;References<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      <td class="Greylink"><a href="#5">5.&nbsp;Contact<img src="{{STATIC_URL}}/images/0.gif" height=1 width=5 border=0><img src="{{STATIC_URL}}/images/arrow.png" border=0><img src="{{STATIC_URL}}/images/0.gif" height=1 width=40 border=0></a><td>
      </tr>
  </table>

  <table>
    <tr>
      <td valign="top" colspan="2">
      <br><br>
      <a name="1"></a>
      <h3>1. Summary</h3><br>
      <table cellspacing=0 cellpadding=0 border=0>
        <tr>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td>
        </tr>
        <tr> <td>
            Pathopred is a pathogenicity predictor for amino acid substitutions, utilising a deep convolutional neural network trained on a large amount of variants in human proteins, incorporating evolutionary information, sequence information, as well as structural information.
          </td>
        </tr>
      </table>

      <br><br>

      <a name="2"></a>
      <h3>2. Usage</h3><br>
      <table cellspacing=0 cellpadding=0 border=0>
        <tr>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td>
        </tr>
        <tr><td>
            <p>
            Sequence input to the server is an amino acid sequence in <a
              href='http://blast.ncbi.nlm.nih.gov/blastcgihelp.shtml'
              target=\'_blank\'>FASTA</a> format. The users can submit 
          queries by either pasting one or several sequences in the text-area
          provided, or, alternatively, uploading a file for batch submission of
          sequences. BLAST will be used to find sequence homologues for each sequence.
            </p>

            <i>Example input:</i> <br>
            <span style="font-family: courier">
              &gt;P26439<br>
              MGWSCLVTGAGGLLGQRIVRLLVEEKELKEIRALDKAFRPELREEFSKLQNRTKLTVLEG<br>
              DILDEPFLKRACQDVSVVIHTACIIDVFGVTHRESIMNVNVKGTQLLLEACVQASVPVFI<br>
              YTSSIEVAGPNSYKEIIQNGHEEEPLENTWPTPYPYSKKLAEKAVLAANGWNLKNGDTLY<br>
              TCALRPTYIYGEGGPFLSASINEALNNNGILSSVGKFSTVNPVYVGNVAWAHILALRALR<br>
              DPKKAPSVRGQFYYISDDTPHQSYDNLNYILSKEFGLRLDSRWSLPLTLMYWIGFLLEVV<br>
              SFLLSPIYSYQPPFNRHTVTLSNSVFTFSYKKAQRDLAYKPLYSWEEAKQKTVEWVGSLV<br>
              DRHKETLKSKTQ<br>
            </span>
            <br>

          <p>
          Variant input to the server is written on the format of reference AA, position, 
          altered AA, e.g. A509G. The variants should be specified below their corresponding sequence identifier starting with >.
          Note that the sequence identifiers should be Uniprot IDs. If you do not have them on this format, you can use the Uniprot <a href="https://www.uniprot.org/uploadlists/">Retrieve/ID mapping tool</a> to prepare them.
          </p>

          <i>Example input:</i> <br>
            <span style="font-family: courier">
              &gt;P26439<br>
              P186L<br>
              P222H<br>
              P222Q<br>
            </span>
            <br>

      </td></tr>
    </table>

    <br><br>

      <a name="3"></a>

      <h3>3. Output</h3><br>
      <table cellspacing=0 cellpadding=0 border=0>
        <tr>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
          <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td></tr>
        <tr><td>
                When the server has finished running a prediction, an output file with predictions will be generated. This file contains a prediction of whether a variant is pathogenic or not, as well as a prediction of whether the variant is severe or not in case of a pathogenic variant. A simple chart of the sequence entropy with the position of the variants marked will also be generated.
                All files can be downloaded as a zip archive file for further analysis.
      </td></tr>
        <tr><td>
                Furthermore, examples of results can be found <a href="/pred/example">here</a>
      </td></tr>
    </table>
    <br><br>

    <a name="4"></a>

    <h3>4. References</h3><br>
    <table cellspacing=0 cellpadding=0 border=0>
      <tr><td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
        <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td>
      </tr>
    {% include "pred/refbib.html" %}
    </table>

  <br><br>
  <a name="5"></a>

    <h3>5. Contact</h3><br
    <table cellspacing=0 cellpadding=0 border=0>
      <tr>
        <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=400></td>
        <td><img src="{{STATIC_URL}}/images/0.gif" border=0 height=1 width=50></td>
      </tr>
      <tr><td>
          <b>Arne Elofsson group</b><br><br>
          <!--<a href="http://www.cbr.su.se/">Center for Biomembrane Research</a><br><br>-->
          <a href="http://www.dbb.su.se/">Department for Biochemistry and Biophysics</a><br>
          The Arrhenius Laboratories for Natural Sciences<br>
          <a href="http://www.su.se/">Stockholm University</a><br>
          SE-106 91 Stockholm, Sweden<br><br>

          <a href="http://www.scilifelab.se">Science for Life Laboratory</a><br>
            Box 1031, 17121 Solna, Sweden
            <br><br>
          <table cellspacing=0 cellpadding=0 border=0>
            <tr><td>E-mail:</td><td>&nbsp;&nbsp;&nbsp;</td><td><a href="mailto:arne@bioinfo.se">arne@bioinfo.se</a></td></tr>
            <tr><td>Phone:</td><td>&nbsp;&nbsp;&nbsp;</td><td>(+46)-8-16&nbsp;4672</td></tr>
            <tr><td>Fax:</td><td>&nbsp;&nbsp;&nbsp;</td><td>(+46)-8-15 3679</td></tr>
          </table>
          <br><br><br>
    </td></tr>
  </table>



{% endblock %}

